src/main/java/com/paneedah/mwc/MWC.java

```diff
@@ -65,6 +62,7 @@ public final class MWC {
     @SidedProxy(serverSide = "com.paneedah.mwc.proxies.CommonProxy", clientSide = "com.paneedah.mwc.proxies.ClientProxy")
     public static CommonProxy commonProxy;
 
+
     @Mod.EventHandler
```

Comment:
Most likely accidental white space
```suggestion
    public static CommonProxy commonProxy;

    @Mod.EventHandler
```